src/elements/content-sidebar/SidebarPanels.js

```diff
@@ -218,6 +205,35 @@ class SidebarPanels extends React.Component<Props> {
                         )}
                     />
                 )}
+                {/* This handles both the default activity sidebar and the activity sidebar with a
+                comment or task deeplink.  */}
+                {hasActivity && (
+                    <Route
+                        exact
+                        path={[
+                            `/${SIDEBAR_VIEW_ACTIVITY}`,
```

Comment:
Can we avoid this array by using optional parameters syntax? Can we use parameter regex to match only comment/task?

```
`/${SIDEBAR_VIEW_ACTIVITY}/:activeFeedItemType(comments|tasks)?/:activeFeedItemId?`
```